[2024-05-29T10:48:55.544+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T10:48:55.564+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v03.transform_data_response manual__2024-05-29T10:37:16.739803+00:00 [queued]>
[2024-05-29T10:48:55.572+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v03.transform_data_response manual__2024-05-29T10:37:16.739803+00:00 [queued]>
[2024-05-29T10:48:55.572+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 9
[2024-05-29T10:48:55.585+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data_response> on 2024-05-29 10:37:16.739803+00:00
[2024-05-29T10:48:55.592+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_weather_workflow_DAG_v03', 'transform_data_response', 'manual__2024-05-29T10:37:16.739803+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpko6f_v09']
[2024-05-29T10:48:55.594+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask transform_data_response
[2024-05-29T10:48:55.595+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=915) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-29T10:48:55.596+0000] {standard_task_runner.py:63} INFO - Started process 917 to run task
[2024-05-29T10:48:55.651+0000] {task_command.py:426} INFO - Running <TaskInstance: my_weather_workflow_DAG_v03.transform_data_response manual__2024-05-29T10:37:16.739803+00:00 [running]> on host cd9caf1c47f0
[2024-05-29T10:48:55.749+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='my_weather_workflow_DAG_v03' AIRFLOW_CTX_TASK_ID='transform_data_response' AIRFLOW_CTX_EXECUTION_DATE='2024-05-29T10:37:16.739803+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-29T10:37:16.739803+00:00'
[2024-05-29T10:48:55.750+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T10:48:55.780+0000] {python.py:237} INFO - Done. Returned value was: weather_of_hcm_29-05-2024
[2024-05-29T10:48:55.781+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T10:48:55.811+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_weather_workflow_DAG_v03, task_id=transform_data_response, run_id=manual__2024-05-29T10:37:16.739803+00:00, execution_date=20240529T103716, start_date=20240529T104855, end_date=20240529T104855
[2024-05-29T10:48:55.851+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T10:48:55.870+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-29T10:48:55.873+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
